src/multibody/geometry.hpp

```diff
@@ -274,100 +268,61 @@ struct GeometryObject
 
     GeometryModel(const se3::Model & model)
       : model(model)
-      , ncollisions(0)
-      , nvisuals(0)
-      , collision_objects()
-      , visual_objects()
+      , ngeoms(0)
+      , geometry_objects()
```

Comment:
@nmansard  Are you talking about the `struct GeometryObject` or the attribute member in GeometryModel `geometry_objects` ?
If we change one, we can change the second accordingly.
Howerver none of the propositions you gave seem totally clear (may be fcl_objects, but it is in fact a wrapper above an fcl::CollisionObject).

GeometryObject/ geometry_objects seems not so bad (as it can be used to represent either collision or visual depending of the use we want to make)